src/components/Display.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Display(props) {
+  const { result } = props;
+  return (
+    <div
+      type="text"
+      name="display"
+      value={result}
+      className="display"
+    />
+  );
+}
+
+Display.propTypes = {
+  result: PropTypes.string.isRequired,
+};
+
```

Comment:
this solves the requirement of value type being `STRING`, but not deafaul value being ZERO.

search on `defaultProps` and implement 
https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png

Reply:
Thank you. I'm not looking at the reviewer docs that must be why :laughing:!

Reply:
I did not implement `defaultProps` as:

```

Display.defaultProps = {
  result: '0',
};
```

Because it isn't allowed with:

```
Display.propTypes = {
  result: PropTypes.string.isRequired,
};
```

See : 

```
propType "result" is required and should not have a defaultProps declaration.eslint(react/require-default-props)
```

Reply:
Yes you are right....it must be provided, but when it's not, the default value is automatically ZERO....like a normal calculator 🙂